### ISSUE TITLE
Any of criteria

### ISSUE THREAD

**Issue by InboundHalo** [char3210/SlimefunAdvancements#24]
Currently there is no way to make an advancement that requires one of a few criteria.

**Comment by AlmostPanda** [comment]
what do you mean by that? you can use more then 1 criteria in advancement

**Comment by InboundHalo** [comment]
I don't think you can make an either or advancement where let's say you could have diamonds in your inventory or gold in your inventory.

**Comment by AlmostPanda** [comment]
the advancement could look like this, but you would have to add group items in your groups.yml
``
diamonds:
  group: items
  display:
    type: DAIMOND
    name: "&bDiamonds &7& &6Gold"
    lore:
      - "&7Get 2 Diamonds and 6 Gold"
  name: "&a[Diamonds & Gold]"
  criteria:
    diamond:
      name: "Have 2 diamonds in inventory"
      type: inventory
      amount: 2
      item:
        type: DIAMOND
    gold:
      name: "Have 6 golds in inventory"
      type: inventory
      amount: 6
      item:
        type: GOLD_INGOT
``

**Comment by InboundHalo** [comment]
I believe that would require both getting diamonds and gold

**Comment by AlmostPanda** [comment]
Oooh you mean 2 different items in 1 criteria, no you cant have that I think so
